### PR TITLE
Adding the troubleshoots notes for miaverse package installation

### PR DESCRIPTION
**Pull request by erawijantari**
I created this based on Mac M1, the troubleshoots for Linux and Windows will be added later.

### CODE REVIEW

**Comment by antagomir** [comment]
Is there a link to this file somewhere in the book?

**Comment by antagomir** [comment]
@erawijantari just to check - can you confirm if there is a link to this file somewhere in the book? Did you check it is visible in the locally rendered version. We can merge after confirmation.

File: Install_MacM1.Rmd [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+	from: 
+	
+	```
+	FLIBS =  -L/opt/R/arm64/gfortran/lib/gcc/aarch64-apple-darwin20.2.0/11.0.0 -L/opt/R/arm64/gfortran/lib -lgfortran -lemutls_w -lm
+	```
+	to*:
+	
+	```
+	FLIBS =  -L/opt/gfortran/lib/gcc/aarch64-apple-darwin20.0/12.2.0 -L/opt/gfortran/lib -lgfortran -lm -lquadmath
+	```
```
Comment (TuomasBorman) [inline Install_MacM1.Rmd:46]: Indentation is off?

File: PackageInstallations_Troubleshoots.Rmd [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+	
+	from: 
+	
+	```
+	FLIBS =  -L/opt/R/arm64/gfortran/lib/gcc/aarch64-apple-darwin20.2.0/11.0.0 -L/opt/R/arm64/gfortran/lib -lgfortran -lemutls_w -lm
+	```
+	to*:
+	
+	```
+	FLIBS =  -L/opt/gfortran/lib/gcc/aarch64-apple-darwin20.0/12.2.0 -L/opt/gfortran/lib -lgfortran -lm -lquadmath
```
Comment (TuomasBorman) [inline PackageInstallations_Troubleshoots.Rmd:48]: Is the indentation correct

**Comment by antagomir** [comment]
@erawijantari can we finalize and close this?

**Comment by erawijantari** [comment]
Yes, I have no further update from my side

**Comment by erawijantari** [comment]
Yes, I have no further update on this.

**Comment by antagomir** [comment]
Thanks!

Can you do the following:

1) Resolve conversations in the above feedback (and thus confirm that they have been taken into account); add comments if necessary, or just close the conversations.

2) Resolve conflicts

3) Once the updated PR goes through successfully, merge it, or ask one of us to merge it.

**Comment by erawijantari** [comment]
I have checked this, and seems that all .Rmd file is now changed to .qmd.
So my change in 06_package.Rmd should be made in 06_package.qmd.
Should I also made the PackageInstallations_Troubleshoots.Rmd into qmd format and made a fresh PR?

**Comment by antagomir** [comment]
Yes, thanks!

**Comment by antagomir** [comment]
Refer to this PR in your new PR so we can followup

**Pull request closed without merging**